src/components/TeamMember/TeamMember.tsx

```diff
@@ -59,14 +57,6 @@ const TeamMember = ({ data }: { data: ITeamMember }) => {
 
       <InfoContainer>
         <MainInfoContainer>
-          <Checkbox
```

Comment:
Instead of removing this checkbox component I think would be better to just add a checked={true} prop to the checkbox. Doing that we can make the component still have the closing and opening feature be kind of disabled.

For now we want to be able to rollback on this  toggling feature if our client want :)